src/api/recordlinker/routes/demo_router.py

```diff
@@ -73,3 +74,69 @@ def get_match_review_records(
     if match_review_record is None:
         raise fastapi.HTTPException(status_code=fastapi.status.HTTP_404_NOT_FOUND)
     return match_review_record
+
```

Comment:
We'll need to update the get_demo_data and get_match_review_records methods to use the cookie data as well.  When the user goes back to the listing page, we want to properly show/filter based on what's been linked/unlinked.